src/classes/NativeLoader.ts

```diff
@@ -0,0 +1,113 @@
+  getAddress(id: string, addressIndex: AddressIndex): AddressInfo;
+  getBalance(id: string): Balance;
+  getNetwork(id: string): string;
+  sync(blockchain: string, id: string): boolean;
+  sign(id: string, psbtBase64: string): string;
+
+  listUnspent(id: string): Array<LocalUtxo>;
+  listTransactions(id: string): Array<TransactionDetails>;
+
+  initAddress(address: string): string;
+  addressToScriptPubkeyHex(id: string): string;
+
+  createTxBuilder(): string;
+  addRecipient(id: string, scriptId: string, amount: number): string;
+  finish(id: string, walletId: string): { base64: string; transactionDetails: any };
+
+  addUnspendable(id: string, outPoint: OutPoint): boolean;
+  addUtxo(id: string, outPoint: OutPoint): boolean;
+  addUtxos(id: string, outPoints: Array<OutPoint>): boolean;
+  doNotSpendChange(id: string): boolean;
+  manuallySelectedOnly(id: string): boolean;
+  onlySpendChange(id: string): boolean;
+  unspendable(id: string, outPoints: Array<OutPoint>): boolean;
+
+  feeRate(id: string, feeRate: number): boolean;
+  feeAbsolute(id: string, feeRate: number): boolean;
+  drainWallet(id: string): boolean;
+  drainTo(id: string, scriptId: string): boolean;
+  enableRbf(id: string): boolean;
+  enableRbfWithSequence(id: string, nsequence: number): boolean;
+  addData(id: string, data: Array<number>): boolean;
+  setRecipients(id: string, recipients: Array<ScriptAmount>): boolean;
+
+  createDescriptor(descriptor: string, network: string): string;
+  descriptorAsString(id: string): string;
+  descriptorAsStringPrivate(id: string): string;
+
+  newBip44(id: string, keychain: KeychainKind, network: Network): string;
+  newBip49(id: string, keychain: KeychainKind, network: Network): string;
+  newBip84(id: string, keychain: KeychainKind, network: Network): string;
+
+  newBip44Public(id: string, fingerprint: string, keychain: KeychainKind, network: Network): string;
+  newBip49Public(id: string, fingerprint: string, keychain: KeychainKind, network: Network): string;
+  newBip84Public(id: string, fingerprint: string, keychain: KeychainKind, network: Network): string;
+
+  combine(psbt64: string, otherPsbt: string): string;
+  extractTx(psbt64: string): string;
+  serialize(psbt64: string): string;
+  txid(psbt64: string): string;
+  feeAmount(psbt64: string): number;
+  psbtFeeRate(psbt64: string): number;
+
+  bumpFeeTxBuilderInit(txid: string, newFeeRate: number): string;
+  bumpFeeTxBuilderAllowShrinking(id: string, address: string): string;
+  bumpFeeTxBuilderEnableRbf(id: string): any;
+  bumpFeeTxBuilderEnableRbfWithSequence(id: string, nsequence: number): any;
+  bumpFeeTxBuilderFinish(id: string, walletId: string): any;
+
+  createTransaction(bytes: Array<number>): string;
+  serializeTransaction(id: string): Array<number>;
+}
+
+export class NativeLoader {
+  protected _bdk: NativeBdkRn = NativeModules.BdkRnModule;
+
+  constructor() {
+    this._bdk = NativeModules.BdkRnModule;
```

Comment:
As commented on the compiled `NativeLoader.js`, the assignment is duplicated

Suggesting to remove it from this line
```suggestion
```